### ISSUE TITLE
Allow Scrolling in Park Graphs

### ISSUE THREAD

**Issue by pizza2004** [OpenRCT2/OpenRCT2#2533]
I would love to be able to go back and forth and see all the data, and possibly choose the length of the graph, to better see what differences there have been over time.

**Comment by pizza2004** [comment]
I was looking at it and I realized that if we wanted to allow you to see more history it would require the new save format and a discussion on how much history we could store.  Personally I think it would be neat to store a complete history assuming it wasn't too space intensive.

**Comment by AaronVanGeffen** [comment]
Yeah, at the moment the data for the graphs are stored in a way that is destructive to the historical data. I'd like that to change, too.
